pages/register/register.controller.js

```diff
@@ -67,7 +68,9 @@
 				self.activeView.credentials = false;
 				self.activeView.terms = true;
 
-				window.grecaptcha.render(document.getElementById('recaptcha'));
+				if (nodeEnv === 'production') {
```

Comment:
for what to ask for the status of `nodeEnv` if you already asked once at `self.enableRecaptcha`? I'd consider changing this to: `if (self.enableRecaptcha) { // render grecaptcha }